broker/src/main/java/org/apache/rocketmq/broker/transaction/queue/TransactionalMessageBridge.java

```diff
@@ -303,9 +303,7 @@ private TopicConfig selectTopicConfig(String topic) {
 
     public boolean writeOp(Integer queueId,Message message) {
         MessageQueue opQueue;
-        if (opQueueMap.containsKey(queueId)) {
-            opQueue = opQueueMap.get(queueId);
-        } else {
+        if ((opQueue = opQueueMap.get(queueId)) == null) {
```

Comment:
Assigning `opQueue` at first may be better than initializing it in `if` condition.

Reply:
Ok, I think this can reduce one query, although the time complexity is only O(1) and I don't think it's a good practice to judge first and then get it. If the opQueueMap has a removal operation, there will be problems.